app/src/main/java/com/davidtomas/taskyapp/features/agenda/data/sync/repository/SyncRepository.kt

```diff
@@ -0,0 +1,110 @@
+package com.davidtomas.taskyapp.features.agenda.data.sync.repository
+
+import android.util.Log
+import com.davidtomas.taskyapp.core.domain._util.Result
+import com.davidtomas.taskyapp.core.domain.model.DataError
+import com.davidtomas.taskyapp.features.agenda.data.event.local.source.EventLocalSource
+import com.davidtomas.taskyapp.features.agenda.data.reminder.local.source.ReminderLocalSource
+import com.davidtomas.taskyapp.features.agenda.data.reminder.remote.api.ReminderService
+import com.davidtomas.taskyapp.features.agenda.data.sync.remote.api.SyncService
+import com.davidtomas.taskyapp.features.agenda.data.task.local.source.TaskLocalSource
+import com.davidtomas.taskyapp.features.agenda.data.task.remote.api.TaskService
+import com.davidtomas.taskyapp.features.agenda.domain.repository.AgendaRepository
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.launch
+
+class SyncRepository(
+    private val syncService: SyncService,
+    private val taskService: TaskService,
+    private val reminderService: ReminderService,
+    private val eventLocalSource: EventLocalSource,
```

Comment:
If you use the naming `localSource` for local items, why pick service for the other side? Wouldn't `remoteSource` be the more consistent approach in that case?